validate.go

```diff
@@ -516,3 +516,45 @@ func (v *Validator) Phone(key, value string, message ...string) {
 		v.Append(key, msg)
 	}
 }
+
+// Range sets the minimum and maximum value of a integer.
+//
+// A maximum of 0 indicates there is no upper limit.
+func (v *Validator) Range(key string, value interface{}, min, max int, message ...string) {
```

Comment:
If you accept `int64` then you don't need to forego type safety and use `interface{}`. Callers with `int` can use `Range("x", int64(v), ...)`. This is the approach most of the stdlib takes (e.g. `strconv`). This also removed the need for the duplicated code below (although that can also be eliminated by converting to int64 in the function).

Reply:
+1, also it's strange `min` and `max` to be `int`. What about if you pass int64 value? The size of `int` type is platform dependent and this can cause problems if you run on 32bit machine. Actuality you will be able to pass `value` that can be out of the `int` range.

Reply:
I don't think we need to overly worry about those kind of problems here, since it's intended for validation of form values. I don't think there's much practical difference between a 32bit or 64bit value here.

Accepting `value int` would probably also be fine. I don't really care, as long as it's not `interface{}`.

Reply:
You're right guys, fixed!